mantidproject/mantid
# Fix calling Load with None returns bad error

Description of work.
This PR fixes an issue with `Load` being called in Python with `Filename=None` passed. Previously it would print a long error message describing the Python container. Now it prints a valid error saying that a `None` is not acceptable.

Additionally I tided `simpleapi` to fix the PEP8 white space issues (which should reduce the number of warnings in flake8) ~~and remove `import os` from the `simpleapi` which was unused into the various scripts which relied on this implicit import.~~ this was too much work so in the next maintenance cycle I'll open another PR to finish this properly. 

**To test:**
Ensure all builds pass
Run the following and check it gives a sane error message
`Load(Filename=None)`

Fixes #19190 .

*Does not need to be in the release notes. - Only noticed internally*

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
